controllers/vdoconfig_controller.go

```diff
@@ -1486,3 +1500,23 @@ func (r *VDOConfigReconciler) deleteCSINamespace(ctx vdocontext.VDOContext) erro
 	}
 	return err
 }
+
+func (r *VDOConfigReconciler) updateCSIConfigmap(ctx vdocontext.VDOContext) error {
+	configKey := types.NamespacedName{
+		Namespace: CsiNamespace,
+		Name:      CSI_FSS_CONFIGMAP,
+	}
+
+	cm := v1.ConfigMap{}
+	err := r.Get(ctx, configKey, &cm)
+	if err != nil {
+		return err
+	}
+
```

Comment:
check for cm.Data != nil 

Suggestion : Rename cm  as it confuses with compatibility matrix in terms of VDO